test-framework/junit5/src/main/java/io/quarkus/test/junit/IntegrationTestUtil.java

```diff
@@ -246,6 +260,33 @@ static Properties readQuarkusArtifactProperties(ExtensionContext context) {
         }
     }
 
+    private static TestLauncher determineTestLauncher() {
+        StackTraceElement[] stackTrace = Thread.currentThread().getStackTrace();
```

Comment:
We don't have a better way to distinguish the build tools here? Looks rather fragile.

Btw, since we are now on Java 11 there is also `java.lang.StackWalker`.

Reply:
PS: It's just for a message, so it's not a big deal.

Reply:
Not at this point where Quarkus has not been launched, hence the hack.

I could have used the StackWalker but I think it's overkill for this very simple case